dhis-2/dhis-web/dhis-web-commons-resources/src/main/webapp/dhis-web-commons/useraccount/updateUserAccountForm.vm

```diff
@@ -17,11 +17,11 @@
     </tr>
     <tr>
         <td><label for="firstName">$i18n.getString( "first_name" ) <em title="$i18n.getString( 'required' )" class="required">*</em></label></td>
-        <td><input type="text" id="firstName" name="firstName" value="$!encoder.htmlEncode( $!userCredentials.userInfo.firstName )"></td>
+        <td><input type="text" id="firstName" name="firstName" value="$!encoder.htmlEncode( $!userCredentials.user.firstName )"></td>

```

Comment:
are you certain this should be changed? not sure if that is a tracker user field

Reply:
Intellij refactor for the win.
I will change it back